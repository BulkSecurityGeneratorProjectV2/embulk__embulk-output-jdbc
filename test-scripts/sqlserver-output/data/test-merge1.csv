ID,CODE,VALUE
2,XXXX,xxx
3,YYYY,yyy
