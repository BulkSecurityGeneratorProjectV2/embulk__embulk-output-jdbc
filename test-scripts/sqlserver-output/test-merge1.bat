osql -d TESTDB -U TEST_USER -P XXXXXXXX -i test-merge.sql
osql -d TESTDB -U TEST_USER -P XXXXXXXX -i test-merge-data.sql

CALL embulk run test-merge1.yml

osql -d TESTDB -U TEST_USER -P XXXXXXXX -Q "SELECT * FROM EMBULK_OUTPUT" > data/temp.txt

echo "diff data/test-merge1_expected.txt data/temp.txt"
diff data/test-merge1_expected.txt data/temp.txt

IF "%ERRORLEVEL%" == "0" (ECHO "OK!") ELSE (ECHO "embulk-output-sqlserver FAILED!")
