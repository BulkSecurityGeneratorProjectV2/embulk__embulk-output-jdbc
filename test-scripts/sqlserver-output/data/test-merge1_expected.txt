 ID          CODE VALUE    
 ----------- ---- -------- 
           1 AAAA aaa      
           2 XXXX xxx      
           3 YYYY yyy      

(3 çsèàóùÇ≥ÇÍÇ‹ÇµÇΩ)
